Tools/machines/lassen-llnl/install_v100_dependencies_toss3.sh

```diff
@@ -119,13 +119,15 @@ python3 -m pip install --upgrade build
 python3 -m pip install --upgrade packaging
 python3 -m pip install --upgrade wheel
 python3 -m pip install --upgrade setuptools[core]
-python3 -m pip install --upgrade cython
-python3 -m pip install --upgrade numpy
+python3 -m pip install --upgrade scikit-build-core
+python3 -m pip install --upgrade "cython>=3"
+python3 -m pip install --upgrade "numpy<2"  # for openPMD-api 0.15.2
 python3 -m pip install --upgrade pandas
 CMAKE_PREFIX_PATH=/usr/lib64:${CMAKE_PREFIX_PATH} python3 -m pip install --upgrade -Ccompile-args="-j10" -Csetup-args=-Dblas=BLAS -Csetup-args=-Dlapack=BLAS scipy
-python3 -m pip install --upgrade mpi4py --no-cache-dir --no-build-isolation --no-binary mpi4py
-python3 -m pip install --upgrade openpmd-api
-CC=mpicc H5PY_SETUP_REQUIRES=0 HDF5_DIR=${SW_DIR}/hdf5-1.14.1.2 HDF5_MPI=ON python3 -m pip install --upgrade h5py --no-cache-dir --no-build-isolation --no-binary h5py
+# Lassen MPI does not suppport MPI v4
+MPI4PY_BUILD_BACKEND="scikit-build-core" python3 -m pip install --upgrade "mpi4py<4" --no-cache-dir --no-build-isolation --no-binary mpi4py
+python3 -m pip install --upgrade openpmd-api==0.15.2
```

Comment:
This line and `numpy<2` can be loosened if https://github.com/openPMD/openPMD-api/pull/1734 works.

Reply:
worked, can  be reset